src/backend/base/langflow/__main__.py

```diff
@@ -439,6 +444,162 @@ def graceful_shutdown(signum, frame):  # noqa: ARG001
             sys.exit(0)
 
 
+@app.command()
+def migration(
+    *,
+    test: bool = typer.Option(False, help="Run migrations in test mode (non-destructive)."),  # noqa: FBT003
+    fix: bool = typer.Option(False, help="Fix migrations (destructive operation)."),  # noqa: FBT003
+    backup: bool = typer.Option(False, help="Backup the SQLite database before migration."),  # noqa: FBT003
+    upgrade: str = typer.Option(None, help="Upgrade the database to the specified revision (e.g., 'head')."),
+    downgrade: str = typer.Option(None, help="Downgrade the database to the specified revision."),
+    force: bool = typer.Option(False, help="Force the operation without confirmation prompts."),  # noqa: FBT003
+) -> None:
+    """Perform migration operations. Only one operation can be specified: test, fix, backup, upgrade, or downgrade.
+
+    If no operation is specified, test migration is assumed.
+
+    Args:
+        test: Run migrations in test mode (non-destructive)
+        fix: Fix migrations (destructive operation)
+        backup: Backup the SQLite database before migration
+        upgrade: Upgrade the database to the specified revision
+        downgrade: Downgrade the database to the specified revision
+        force: Force the operation without confirmation prompts
+    """
+    settings_service = get_settings_service()
+    db_service = get_db_service()
+
+    # Determine which operation was requested
+    operations = {"test": test, "fix": fix, "backup": backup, "upgrade": bool(upgrade), "downgrade": bool(downgrade)}
+
+    # Get list of requested operations
+    requested_ops = [op for op, value in operations.items() if value]
+
+    # Default to test mode if no operation specified
+    if not requested_ops:
+        operations["test"] = True
+        requested_ops = ["test"]
+
+    # Validate only one operation requested
+    if len(requested_ops) > 1:
+        typer.echo("Please specify exactly one migration operation: test, fix, backup, upgrade, or downgrade.")
+        raise typer.Exit(code=1)
+
+    operation = requested_ops[0]
+
+    # For destructive operations, backup first if using SQLite
+    if (
+        not force
+        and operation in ["fix", "upgrade", "downgrade"]
+        and settings_service.settings.database_url.startswith("sqlite")
+    ):
+        if not typer.confirm("Would you like to backup the database before proceeding?"):
+            if not typer.confirm("Are you sure you want to proceed without a backup?"):
+                raise typer.Abort()  # noqa: RSE102
```

Comment:
```suggestion
                raise typer.Abort

Reply:
https://typer.tiangolo.com/tutorial/terminating/#abort

It should be Abort() according to the docs

Reply:
You don't need the parenthesis.
Try can try the doc example without the parenthesis, it works fine.